apps/website/src/pages/docs/anchor.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Anchor
-description:
+description: The anchor component works like a simple hyperlink, targeting a specific URL. Additionally, it can also target a named location within a page. For example, you might want to target a content area that appears below the fold so the user does not have to scroll to discover it.
```

Comment:
This is a bit too wordy. Descriptions need to be short enough to fit in 3 small lines
![](https://user-images.githubusercontent.com/9084735/221204201-8c6519ab-1cab-41e0-b9b4-d9ddc0e3f34b.png)

Reply:
Valid. Didn't realize that the "Description" field is used in the tile synopsis. There are plenty reasons to provide more clarifying text (I think "wordy" might be a bit harsh ;) ), but I will include extra information about the component below the first live example and not in the "Description" field. First timer here.

Reply:
Definitely, there is a lot to document even for a very simple component like this one (for instance, anchors can link to not just pages but also to files). If you would like to further expand on the description, feel free to do so before the paragraph you just added.